python-pro-data-1/zakladni-dotazy/zakladni-dotazy.md

```diff
@@ -35,8 +38,11 @@ Index(['Afghanistan', 'Åland Islands', 'Albania', 'Algeria', 'American Samoa',
 
 Podívejme se nejprve, jaké informace jsou v naší tabulce obsažené.
 
-```pycon
->>> staty.info()
+```py
+staty.info()
```

Comment:
```suggestion
print(staty.info())
```

Reply:
Ne, zkousel jsem to. Opravdu to vypise bez `print()`.

Reply:
No jo, to jsem vlastně řešila i v kurzu. Vrací to `None`. Je to trochu zvláštní metoda. Dík 🙂